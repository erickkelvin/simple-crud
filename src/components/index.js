import List from './List/List.js';
import ListItem from './ListItem/ListItem.js';
import Form from './Form/Form.js';
import Field from './Field/Field.js';
import Button from './Button/Button.js';
import Spinner from './Spinner/Spinner.js';
import NotFound from './NotFound/NotFound.js';

export {
  List,
  ListItem,
  Form,
  Field,
  Button,
  Spinner,
  NotFound
};