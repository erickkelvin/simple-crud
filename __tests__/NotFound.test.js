import { NotFound } from 'components';

describe('NotFound test', () => {
  const notFound = new NotFound();

  it('render', () => {
    expect(notFound.render()).toMatchSnapshot();
  });
});