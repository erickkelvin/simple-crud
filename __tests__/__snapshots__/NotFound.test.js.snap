// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound test render 1`] = `
"
      <div id='notFoundContainer' class='pageContainer'>
        <h3>Erro 404</h3>
        <div>
          Não foi possível encontrar a página requisitada.
        </div>
      </div>
    "
`;
